src/server/scripts/EasternKingdoms/Karazhan/boss_prince_malchezaar.cpp

```diff
@@ -421,6 +421,7 @@ class boss_malchezaar : public CreatureScript
 
                     if (target)
                         DoCast(target, SPELL_AMPLIFY_DAMAGE);
+                        AmplifyDamageTimer = urand(20000, 30000);
```

Comment:
either the code indentation is wrong or you have an issue in your code: the way it is indented, it looks like the `AmplifyDamageTimer` should be part of the `if (target`) block, but it isn't because there are no { brackets }.

So it should be either:

```
if (target)
    DoCast(...);
AmplifyDamageTimer....;
```
or:

```
if (target)
{
    DoCast(...);
    AmplifyDamageTimer....;
}

```
In the first case you DO NOT WANT to include it in the if block, in the second case you DO WANT to include it in the if.